--- Classes/ContentObject/ContentContentObject.old.php	2023-05-10 16:15:03.712611263 +0700
+++ Classes/ContentObject/ContentContentObject.php	2023-05-10 16:16:16.007091114 +0700
@@ -85,7 +85,8 @@
                         $_procObj->modifyDBRow($row, $conf['table']);
                     }
                     $registerField = $conf['table'] . ':' . ($row['uid'] ?? 0);
-                    if (!($frontendController->recordRegister[$registerField] ?? false)) {
+                    // $registered = $frontendController->recordRegister[$registerField] ?? false;
+                    if ($registerField) {
                         $this->cObj->currentRecordNumber++;
                         $cObj->parentRecordNumber = $this->cObj->currentRecordNumber;
                         $frontendController->currentRecord = $registerField;
